docs/How-To/cli/README.md

```diff
@@ -102,27 +111,31 @@ Usage
 create-batch-auction [file]
 ```
 
+| **Argument** |  **Description**                                             |
+| :----------- | :----------------------------------------------------------- |
+| file         | file that contains required fields for a fixed price auction |
+
 Field description of the input file
 
-| **Field**         |  **Description**                                                                    |
-| :---------------- | :---------------------------------------------------------------------------------- |
-| allowed_bidders | The list of allowed bidders that can participate in the auction, with a maximum possible bid amount for each bidder. It is empty when an auction is created. The module is designed to delegate permission to an external module to add its allowed bidders to the auction. |
-| start_price       | The starting price of the selling coin; it is proportional to the paying coin denom. This is the matched price. | 
-|min_bid_price | The minimum bid price that bidders must place with.|
-| selling_coin      | The selling amount of coin for the auction                                      | 
-| paying_coin_denom | The paying coin denom that bidders use to bid with                                  | 
-| vesting_schedules | The vesting schedules that release the paying coins to the autioneer                | 
-| start_time        | The start time of the auction                                                       | 
-| end_times          | The list of the end times of the auction in consideration of the extended rounds                                                         | 
-| max_extended_round   | The maximum number of extended rounds that provides additional opportunity for the bidders to place bids when more than a certain ratio of the number of the matched bids are reduced compared to the previous end time  |
+| **Field**           |  **Description**                                                                    |
+| :------------------ | :---------------------------------------------------------------------------------- |
+| start_price         | The starting price of the selling coin; it is proportional to the paying coin denom. This is the matched price | 
```

Comment:
```suggestion
| start_price         | The starting price of the selling coin; it is proportional to the paying coin denom. This is the matched price | 
```

Since this is not the matched price, how about modifying like the following?
```suggestion
| start_price         | The starting price of the selling coin; it is proportional to the paying coin denom. This might not be the matched price and this can be used as a guideline of the price from the auctioneer. | 
```

Reply:
Yes, `This is the matched price` is incorrect description. I will update it with the suggested change.